example/snake/actions.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <boost/variant/variant.hpp>
```

Comment:
Neote: I'm using boost variant over std variant, as std variant depends on the macOS 10.14 runtime (for the exception definition used in std::visit). I wrote this on macOS 10.13.